spec/rubocop/cop/rspec/example_length_spec.rb

```diff
@@ -0,0 +1,63 @@
+# encoding: utf-8
+
+describe RuboCop::Cop::RSpec::ExampleLength, :config do
+  subject(:cop) { described_class.new(config) }
+  let(:cop_config) do
+    {
+      'Max' => 3
+    }
+  end
+
+  it 'allows an empty example' do
+    inspect_source(cop, ['it do',
```

Comment:
Format inspect_source like in other examples, see https://github.com/nevir/rubocop-rspec/commit/bd5fe5997de2d716755033b12df89cb66d9731c2 for reference